packages/devtools_app/test/network_controller_test.dart

```diff
@@ -95,7 +95,7 @@ void main() {
       const numRequests = 16;
 
       // Refresh network data and ensure requests are populated.
-      await controller.networkService.refreshNetworkData();
+      await controller.networkService!.refreshNetworkData();
```

Comment:
network service should not be nullable. See Ben's PR here: https://github.com/flutter/devtools/pull/3880/files#diff-5823db4661b7010348a2cbb6910f27b950c8cc3b63b21744bce8fe5c779b47d8

Reply:
done